pkg/env/env.go

```diff
@@ -343,8 +349,12 @@ func (e *testEnv) execFeature(ctx context.Context, t *testing.T, f types.Feature
 		// assessments run as feature/assessment sub level
 		assessments := features.GetStepsByLevel(f.Steps(), types.LevelAssess)
 
-		for _, assess := range assessments {
-			t.Run(assess.Name(), func(t *testing.T) {
+		for i, assess := range assessments {
+			assessName := assess.Name()
+			if assessName == "" {
+				assessName = fmt.Sprintf("%s-%d", t.Name(), i)
```

Comment:
Similarly, I would drop `t.Name` here and just use `i` to avoid repetition.
```suggestion
				assessName = fmt.Sprintf("%d", i)
```
or some prefix:

```suggestion
				assessName = fmt.Sprintf("assess-%d",i)
```

Reply:
Likewise, great suggestion.